src/Interfaces/hamiltonians.jl

```diff
@@ -331,10 +355,15 @@ julia> h = offdiagonals(H, address)
 ```
 Part of the [`AbstractHamiltonian`](@ref) interface.
 
-# Extemded help
+# Extended help
+
+`offdiagonals(h, address)` returns an iterator of type `<:AbstractOffdiagonals`. It defaults
+to returning `Offdiagonals(h, a)`.
 
-`offdiagonals` return and iterator of type `<:AbstractOffdiagonals`. It defaults to
-returning `Offdiagonals(h, a)`
+`offdiagonals(column)` returns `offdiagonals(column.operator, column.address)` if `column`
+is a default [`OperatorColumn`](@ref). Otherwise, it returns an iterator over non-zero
+off-diagonal elements of the column as pairs, `newaddress => matrixelement`, and is not
+necessarily a vector.
 
 See also [`Offdiagonals`](@ref Main.Hamiltonians.Offdiagonals),
 [`AbstractOffdiagonals`](@ref Main.Hamiltonians.AbstractOffdiagonals).
```

Comment:
I would suggest to delete the Extended Help section altogether. The information here is either not fully true (E.g. `HOCartesianContactInteractions` does not return a subtype of `AbstractOffdiagonals`), or related to implementation details that we will likely change soon anyway.